app/controllers/documents_controller.rb

```diff
@@ -38,6 +38,9 @@ def destroy
 
   def vectorize
     Mediator.ingest(@document)
+  rescue Mediator::IngestError
+    flash[:error] = ["Unable to ingest document. Likely an unsupported format."]
```

Comment:
Any reason you're not capturing the exception as `=> e` and rendering its message?

Reply:
Rushing. 😄  Fixed.